lxd/storage/drivers/driver_pure.go

```diff
@@ -121,6 +127,24 @@ func (d *pure) Create() error {
 		return err
 	}
 
+	revert := revert.New()
+	defer revert.Fail()
+
+	poolSizeBytes, err := units.ParseByteSizeString(d.config["size"])
+	if err != nil {
+		return fmt.Errorf("Failed to parse storage size (quota limit): %w", err)
```

Comment:
im interested to know why you mentioned "quota" in this message, is this a pure specific concept?

Reply:
This makes no sense since the value is parsed from config. Will fix it.